tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -266,8 +251,61 @@ def _verify_built_artifact(self, build_dir, function_logical_id, expected_files,
 
         self.assertTrue(any([True if self.EXPECTED_RUBY_GEM in gem else False for gem in os.listdir(str(gem_path))]))
 
-    def _verify_resource_property(self, template_path, logical_id, property, expected_value):
 
-        with open(template_path, 'r') as fp:
-            template_dict = yaml_parse(fp.read())
-            self.assertEquals(expected_value, template_dict["Resources"][logical_id]["Properties"][property])
+class TestBuildCommand_GoFunctions(BuildIntegBase):
+
+    EXPECTED_FILES_GLOBAL_MANIFEST = set()
+    EXPECTED_FILES_PROJECT_MANIFEST = {'main.handler'}
+    EXPECTED_BINARY = 'main.handler'
+
+    FUNCTION_LOGICAL_ID = "Function"
+
+    @parameterized.expand([
+        ("go1.x", False),
+        ("go1.x", "use_container")
+    ])
+    def test_with_default_go_mod(self, runtime, use_container):
+        overrides = {"Runtime": runtime, "CodeUri": "Go"}
+        cmdlist = self.get_command_list(use_container=use_container,
+                                        parameter_overrides=overrides,
+                                        # TODO remove this once aws-lambda-builders is released
```

Comment:
So this is a problem I kept running into. Even if you skip pull images, this code won't build on Travis because there are no containers there. Instead, I would just turn off Container builds in the here until the Docker container is released

Reply:
This was for me locally. I wanted to remove it only after we got closer to be able to merge this but didn't want to manage/keep track of this locally.

Not sure we want to turn off this test. We run the risk of needing to do a release and making the build unavailable with the `--use-container` flag. I would prefer we either leave it until we release `aws-lambda-builders` and containers get updated or we only ignore this test when running travis in the develop branch and all tests run when merging to master.

Reply:
The other thing is until `aws-lambda-builders` is released `sam build` for a go function won't work because the released `aws-lambda-builders` does not have the Go Modules builder.